machine_learning/local_weighted_learning/local_weighted_learning.py

```diff
@@ -71,6 +71,10 @@ def local_weight_regression(
 def load_data(dataset_name: str, cola_name: str, colb_name: str) -> np.mat:
     """
     Function used for loading data from the seaborn splitting into x and y points
+    >>> np.ones((3,2))*np.ones((1,2))
+    array([[1., 1.],
+           [1., 1.],
+           [1., 1.]])
```

Comment:
```suggestion
```
Let's remove the doctest from this function because pytest will run the function and will try to import seaborn.